sky/resources.py

```diff
@@ -196,8 +194,15 @@ def instance_type(self):
         return self._instance_type
 
     @property
-    def cpu(self) -> Optional[str]:
-        return self._cpu
+    def cpus(self) -> Optional[str]:
+        """Returns the number of vcpus that each instance must have.
+
+        For example, cpus='4' means each instance must have exactly 4 vcpus,
+        and cpus='4+' means each instance must have at least 4 vcpus.
```

Comment:
```suggestion
        and cpus='4+' means each instance must have at least 4 vcpus.
        
```

(styleguide) break paragraphs by blank lines

Reply:
Good catch. Fixed.